plans/placebo/go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/ipfs/testground/plans/placebo
 
 go 1.14
 
-require github.com/ipfs/testground/sdk/runtime v0.3.0
+require (
+	github.com/ipfs/testground/sdk/runtime v0.4.0
+	github.com/pkg/errors v0.9.1 // indirect
+	golang.org/x/tools v0.0.0-20200227222343-706bc42d1f0d // indirect
+)
```

Comment:
go mod seems broken; I don't know how this happened. If you remove the indirect dependencies and run go mod tidy, it doesn't add them 🤷‍♂️

Reply:
It was happening on the `plans` for me as well. I am on Go 1.13.1, that might be the case, will upgrade now.